draft-ietf-rats-architecture.md

```diff
@@ -296,56 +296,62 @@ each router is an Attester and the main router is the Lead Attester.
-| |  '----------------'  Collecting      | (via Internal Links or  |
-| |                      Claims          | Network Connections)    |
-| |                                      |                         |
-| | Lead Attester A                      |                         |
-| '--------------------------------------'                         |
-|                                                                  |
-|                       Composite Device                           |
-'------------------------------------------------------------------'
+              .-----------------------------.
+              |       Remote Verifier       |
+              '-----------------------------'
+                             ^
+                             | Evidence of
+                             | Composite Device
+                             |
+.----------------------------|----------------------------------.
+| .--------------------------|--------.                         |
+| |                   .-------------. |                         |
+| | Collecting Claims | Attesting   | |                         |
+| |         /-------->| Environment | |                         |
+| |         |         '-------------' |      .------------.     |
+| | .----------------.                |      |            |     |
+| | | Target         |                |<-----| Attester B |-.   |
+| | | Environment(s) |                |      '------------' |   |
+| | | (Optional      |                |<-------| Attester C |-. |
+| | |  Local         |                |        '------------' | |
+| | |  Verifier)     |                |<---------| ...        | |
+| | '----------------'                | Evidence '------------' |
+| |                                   | of Attesters            |
+| | Lead Attester A                   | (via Internal Links or  |
+| '-----------------------------------'  Network Connections)   |
+|                                                               |
+|                       Composite Device                        |
+'---------------------------------------------------------------'
 ~~~~
 {: #composite title="Conceptual Data Flow for a Composite Device"}
 
 In the Composite Device, each Attester generates its own Evidence by its
 Attesting Environment(s) collecting the claims from its Target Environment(s).
 The Lead Attester collects the Evidence of all other Attesters and then
-generates the Evidence of the whole Composite Attester.
-
-The Lead Attester's Attesting Environment may or may not include its own
-Verifier.
-One situation is that the Attesting Environment has no local Verifier.
-In this situation, the Lead Attesting Environment simply combines the various
+generates the Evidence of the whole Composite Device.
+
+Inside the entity of Attester, there may be a Target Environment
+that has the ability to verify the Evidence of other Attesters,
+it can be seen as an entity of Verifier and be called a local Verifier.
```

Comment:
Maybe let me attempt a rephrase:
"The Entity that takes on the role of a Attester myay also include the role of a Verifier in its Target Environment. A Verifier residing in a Target Environment of an Attester is called a Local Verifier."

Reply:
It's better, I'll change the text.